README.md

```diff
@@ -4,68 +4,95 @@ languages:
 - csharp
 products:
 - azure
+- azure-storage
+- dotnet-core
 description: "This repo contains sample code demonstrating the backup/restore and recoverable deletion functionality of Azure Key Vault using the Azure .NET SDK."
-urlFragment: net-sdk-samples
+urlFragment: key-vault-dotnet-recovery
 ---
 
-# .NET SDK samples for recovering and restoring Azure Key Vault entities 
+# How to recover and restore Azure Key Vault entities with .NET 
 
-This repo contains sample code demonstrating the backup/restore and recoverable deletion functionality of Azure Key Vault using the [Azure .Net SDK](https://docs.microsoft.com/dotnet/api/overview/azure/key-vault?view=azure-dotnet). The scenarios covered by these samples include:
+## Prerequisites
 
-* Backing up and restoring Key Vault secrets and keys
-* Enabling recoverable deletion on creating a new vault
-* Enabling recoverable deletion on an existing vault
-* Recovering or permanently deleting deleted vaults
-* Recovering or permanently deleting Key Vault secrets, keys, and certificates
+To complete this tutorial:
 
-The recoverable deletion functionality is also referred to as 'soft delete'; consequently, a permanent, irrecoverable deletion is referred to as 'purge'.
+* Install .NET Core 2.0 version for [Linux] or [Windows]
```

Comment:
2.0 is out of support. 2.1 is LTS, but so is 3.1. We should at least recommend an LTS.